prep.sh

```diff
@@ -41,7 +41,7 @@ for LINE in $LINEZ; do
 done
 
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
-version=1152.vafc19b26d5aa # TODO https://github.com/jenkinsci/plugin-compat-tester/pull/345
+version=1178.vbef3c43d0e69
```

Comment:
One of the most significant lines of this patch: the use of a release build of PCT.

Reply:
Do you happen to know what obviated the need for https://github.com/jenkinsci/plugin-compat-tester/commit/ef1d7a25bd939ddeef7bc448ea4f8cb06b4a577e? `useUpperBounds`? Would be happy to close https://github.com/jenkinsci/plugin-compat-tester/pull/345.

Reply:
I have no idea and I am scared to find out.